modules/ROOT/pages/manage-versions-instances-concept.adoc

```diff
@@ -3,9 +3,19 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-After you create an API in Anypoint Platform, you configure the version details for the API and then publish it to Anypoint Exchange (Exchange). The API version resides in Exchange, and not in API Manager. However, you then manage this API version from API Manager.
+After an API is created, you can configure version details for the API and publish them to Anypoint Exchange (Exchange). The API asset version resides in Exchange, and not in API Manager. However, you manage the version of an API instance from API Manager. You can create multiple instances and expand the version information to view details about them.
```

Comment:
Also we could expand this part in more details:
```
However, you manage the version of an API instance from API Manager
```
Something like:
```
However, once you [add a new API Instance in API Manager](https://docs.mulesoft.com/api-manager/2.x/create-instance-task), you can change the API Asset version of that API Instance.
```

Reply:
I link to exchange in the bottom.